internal/provider/vsphere/installConfig.go

```diff
@@ -5,31 +5,17 @@ import (
 
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/installcfg"
-	"github.com/openshift/assisted-service/models"
 )
 
-func setPlatformValues(platform *installcfg.VsphereInstallConfigPlatform, clusterPlatform *models.VspherePlatform) {
-	if clusterPlatform != nil && clusterPlatform.VCenter != nil {
-		platform.VCenter = *clusterPlatform.VCenter
-		platform.Username = *clusterPlatform.Username
-		platform.Password = *clusterPlatform.Password
-		platform.Datacenter = *clusterPlatform.Datacenter
-		platform.DefaultDatastore = *clusterPlatform.DefaultDatastore
-		platform.Network = *clusterPlatform.Network
-		platform.Cluster = *clusterPlatform.Cluster
-		if clusterPlatform.Folder != nil {
-			platform.Folder = *clusterPlatform.Folder
-		}
-	} else {
-		// This is to support adding parameters in day2
-		platform.Cluster = PhCluster
-		platform.VCenter = PhVcenter
-		platform.Network = PhNetwork
-		platform.DefaultDatastore = PhDefaultDatastore
-		platform.Username = PhUsername
-		platform.Password = PhPassword
-		platform.Datacenter = PhDatacenter
-	}
+func setPlatformValues(platform *installcfg.VsphereInstallConfigPlatform) {
```

Comment:
hm... why do we need it? We should ask for creds neither.

Reply:
It's just a placeholders

Reply:
sorry, thought that it's for day2